.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches:
+      - '**'
+
+
+jobs:
+  run_tests:
+    name: tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
```

Comment:
If you aren't going to use more than one OS, it's simpler to omit this `os:` key, and just put `"ubuntu-20.04"` in the `runs-on:` value.